core/handler/device/store.go

```diff
@@ -122,6 +130,7 @@ func (s *RedisDeviceStore) Set(new *Device, properties ...string) (err error) {
 	if new.old == nil {
 		new.CreatedAt = now
 	}
+	s.builtinFilter(new)
```

Comment:
Yeah definitely rename it to attributes, `builtinFilter` does not make _any_ sense